src/Command/BaseCommand.php

```diff
@@ -14,15 +14,40 @@
 namespace Sonata\BlockBundle\Command;
 
 use Sonata\BlockBundle\Block\BlockServiceManagerInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 
-abstract class BaseCommand extends ContainerAwareCommand
+abstract class BaseCommand extends Command
 {
+    /**
+     * @var BlockServiceManagerInterface
+     */
+    protected $blockManager;
```

Comment:
Why not `private`?

Reply:
Because this property must be accessible from `DebugBlocksCommand`.